### ISSUE TITLE
Certbot on aws lambda?

### ISSUE THREAD

**Issue by tommedema** [Miserlou/lambda-packages#69]
I would like to contribute to this repo by building certbot for AWS lambda.

Am I correct in assuming that I would have to spin up a AWS linux EC2 instance, and then build certbot in a statically linked manner? How would one do this? I would like to give it a try after being brought into the right direction.

**Comment by Miserlou** [comment]
Yep, either from a docker. But isn't certbot in pure python already?

**Comment by tommedema** [comment]
@Miserlou yes but I'm trying to get an executable that I could run from e.g. node.js

**Comment by Miserlou** [comment]
If certbot is already pure python, you don't need to "run it from node.js". I'd suggest packaging and deploying using Zappa, then use the `invoke` command to call something like `subprocess.check_output([ "/usr/bin/python2.7", "/var/task/certbot/whatever/certbots/cli/main/entrypoint/is"` etc
